packages/xo-web/src/xo-app/proxies/deploy-proxy.js

```diff
@@ -71,12 +81,18 @@ const Modal = decorate([
     <Container>
       <SingleLineRow>
         <Col mediumSize={4}>
-          <Label htmlFor={state.idSelectSr}>{_('destinationSR')}</Label>
+          <Label htmlFor={state.idSelectSr}>
+            {_('destinationSR')}{' '}
+            <Tooltip content={_('proxySrPredicateInfo')}>
```

Comment:
The `select` contains the SRs at least connected to a HVM-capable host.
Why do you add this?

Reply:
It can contains SRs which are not connected to an HVM-capable host.

Reply:
But you can say that the selector only contains the SRs at least connected to a HVM-capable host, not 'the SR should be....`,  because when we read this, we think that the selector contains others SRs that are not connected to an HVM-capable host.

Reply:
You're right :+1:  I thought the comment is for the tooltip usage.